src/api/app/views/webui/request/_review_by.html.haml

```diff
@@ -0,0 +1,30 @@
+%li
```

Comment:
This looks like it's a View component, something like `BsRequestReviewComponent`. Then all those if's below can live inside the component class and the view becomes waay dumber

Reply:
The view has been updated and now it's a pretty simple view. I think we don't need a view component for this now.